qa/tasks/mgr/test_progress.py

```diff
@@ -112,28 +112,28 @@ def _simulate_failure(self, osd_ids=None):
         
         return ev
 
-    def _simulate_back_in(self,osd_ids, ev1):
-
+    def _simulate_back_in(self, osd_ids=None, initial_event):
```

Comment:
also, what's the point of having a default parameter in the middle of parameter list? another reason to pass `[0]` explicitly.

Reply:
@tchaikov
I was following the convention used in _simulate_failure, but yea you’re right I guess you could just make [0] default parameter instead

Reply:
```
>>> def foo(a, b=0, c):
...   pass
...
  File "<stdin>", line 1
SyntaxError: non-default argument follows default argument
```

also, please note, we should never pass an array as the default parameter. you could try to play around with
```py
def append(number, number_list=[]):
    number_list.append(number)
    print(number_list)
    return number_list
```

Reply:
@tchaikov 
I see what you mean now, sorry I think I misunderstood what you mean by explicitly passing variable